tools/Makefile

```diff
@@ -1,12 +1,12 @@
 export OPENAPI_FOLDER=../openapi
 export OPENAPI_FILE_NAME=atlas-api.yaml
-export SDK_FOLDER=../api/v1alpha
+export SDK_FOLDER=../mongodbatlasv2/latest
 
 .DEFAULT_GOAL := generate_client
 
 .PHONY: clean_client
 clean_client:
-	rm -Rf ../api/v1alpha
+	rm -Rf ../mongodbatlasv2/latest
```

Comment:
```suggestion
	rm -Rf $(SDK_FOLDER)
```

Reply:
I would prefer to not apply this change as relying on env var specifically introduces some risks (which happened to some upstream projects removing users hard drive)

Reply:
note that this is not an env var per se it is a makefile var, I'm ok either way though